cloud/google/clients/computeservice.go

```diff
@@ -12,6 +25,19 @@ type ComputeService struct {
 	service *compute.Service
 }
 
+type GCEClientComputeService interface {
+	ImagesGet(project string, image string) (*compute.Image, error)
+	ImagesGetFromFamily(project string, family string) (*compute.Image, error)
+	InstancesDelete(project string, zone string, targetInstance string) (*compute.Operation, error)
+	InstancesGet(project string, zone string, instance string) (*compute.Instance, error)
+	InstancesInsert(project string, zone string, instance *compute.Instance) (*compute.Operation, error)
+	ZoneOperationsGet(project string, zone string, operation string) (*compute.Operation, error)
```

Comment:
should GlobalOperationsGet be in the interface?

Reply:
Added, thanks!